### DISCUSSION TITLE
3D Messen - Darstellung (Textur) - Einträge entfernen

### DISCUSSION THREAD

**Discussion by faulgurke** [e-netze/webgis-community#58] [Q&A]
Ist es möglich beim Werkzeug 3D Messen die Einträge "Luftbild" und "Luftbild mit Straßennamen" aus der Liste der Texturen zu entfernen?
"Monochrome" und "aktuelle Kartendarstellung" würden auch schon genügen.

**Comment by gis-eni** [comment]
Warum sollte man die Luftbild Oberfläche nicht anbieten. Genau das verwenden viele, weil es am ehesten einer realistischen 3D Ansicht entspricht.
